enterprise/server/backends/gcs_cache/gcs_cache.go

```diff
@@ -93,6 +104,8 @@ func (g *GCSCache) setBucketTTL(ctx context.Context, bucketName string, ageInDay
 			},
 		},
 	}
+	_, spn = tracing.StartSpan(ctx)
+	defer spn.End()
```

Comment:
I think possibly we have two spans happening inside this function -- since they are named (automagically) after the calling function, you may want to name these two explicitly?

	ctx, spn = tracing.StartSpan(ctx, trace.WithAttributes(... somehow you put name in here....))
	defer spn.End()

Reply:
(maybe doublecheck this isn't happening elsewhere too, 2 spans 1 func)

Reply:
Done!